devito/ops/operator.py

```diff
@@ -73,7 +73,7 @@ def _specialize_iet(self, iet, **kwargs):
             if f.is_Constant:
                 continue
 
-            pre_time_loop.extend(create_ops_dat(f, name_to_ops_dat, ops_block))
+            pre_time_loop.extend(list(create_ops_dat(f, name_to_ops_dat, ops_block)))
```

Comment:
This is the only part that i didn't like, but for the sake of the rest of  the code, I think its better.